### ISSUE TITLE
Increase code coverage

### ISSUE THREAD

**Issue by fabianp** [openopt/chop#21]
Should be around 90%, currently its 67%

**Comment by GeoffNN** [comment]
Should we exclude files like `data_utils.py` from the coverage calculation?

**Comment by fabianp** [comment]
yes, you can exclude things that we don't want to test, like the data loading.
